grr/server/grr_response_server/gui/api_plugins/timeline_test.py

```diff
@@ -206,6 +206,83 @@ def testRaisesOnIncorrectFlowType(self):
     with self.assertRaises(ValueError):
       self.handler.Handle(args)
 
```

Comment:
There should also be a test case with incorrect format.

Reply:
I'll add it. Thank you!